packages/react-components/lib/beacons/beacon-table-datagrid.tsx

```diff
@@ -0,0 +1,132 @@
+import { ApiServerModelsTortoiseModelsBeaconsBeaconStateLeaf as BeaconState } from 'api-client';
+import { DataGrid, GridColDef, GridValueGetterParams, GridCellParams } from '@mui/x-data-grid';
+import { Box, SxProps, Typography, useTheme } from '@mui/material';
+import * as React from 'react';
```

Comment:
nit: Is there any reason to do this instead of how we do it in the rest of the files (import React from 'react';)?
In fact, we could start by doing import {useState, useEffect, ...} from React to be more specific.

Reply:
Looks like I copied over from doors and lifts too much, thanks for catching that! https://github.com/open-rmf/rmf-web/pull/733/commits/7cd343db68680249abbc18a01ca85faf46667d94 fixes it as well as for doors and lifts